src/sass/atoms/_shame.scss

```diff
@@ -0,0 +1,11 @@
+// refactor to use the new clearfix? + we shouldn't even need one?
+.clearfix {
```

Comment:
This needs to be documented/commented in the readme.md, since it's not included by default in Once.